src/docfx/build/xref/XrefMap.cs

```diff
@@ -14,18 +14,52 @@ namespace Microsoft.Docs.Build
     internal class XrefMap
     {
         // TODO: key could be uid+moniker+locale
-        private readonly IReadOnlyDictionary<string, Lazy<(List<Error>, XrefSpec)>> _internalXrefMap;
+        private readonly IReadOnlyDictionary<string, List<Lazy<(List<Error>, XrefSpec)>>> _internalXrefMap;
```

Comment:
>List [](start = 53, length = 4)

can we make this a `sortset`, so we don't have to sort it every time.

Reply:
Not really. Because with `SortedSet`, every time you add an item to it, there will be a sorting execution. With list, you can sort it once in the end.

Reply:
This depends on how we use the list, if we need to sort it everytime accessing it, then we should sort it while adding.

Reply:
maybe we just need to sort it after we build the xrefmap.

---
In reply to: [234477727](https://github.com/dotnet/docfx/pull/3629#discussion_r234477727) [](ancestors = 234477727)